src/navigation/main-navigator.tsx

```diff
@@ -1,12 +1,14 @@
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import * as React from 'react';
 
+import { PaymentScreen } from '../modules/payment/screens/payment-screen';
 import { DetailsScreen } from '../modules/products/screens/detail-products/details-screen';
 import { TabNavigator } from './tab-navigator';
 
```

Comment:
also here you could said: 
`export enum MainStackScreens {
  'Tab' = 'Tab',
  'Details' = 'Details',
}` and `const { Tab, Details } = MainStackScreens` and use that keys

Reply:
Done.